api/src/main/java/org/pmiops/workbench/rdr/RdrMapper.java

```diff
@@ -35,14 +38,23 @@
   // Workspace User will be populated by a call to FireCloud
   // This will be handle in ServiceImpl Class
   @Mapping(target = "workspaceUsers", ignore = true)
-  RdrWorkspace toRdrModel(DbWorkspace employeeDbEntity);
+  RdrWorkspace toRdrModel(DbWorkspace dbWorkspace);
 
   ZoneOffset offset = OffsetDateTime.now().getOffset();
 
-  default OffsetDateTime toModelOffsetTime(Timestamp dbTime) {
+  default OffsetDateTime toModelOffsetTime(@Nullable Timestamp dbTime) {
+    if (dbTime == null) {
```

Comment:
Unrelated, but better to do things this way, and it caused my tests to fail.

Reply:
ok